test/utils/rest.request.ts

```diff
@@ -37,3 +37,23 @@ export const restRequestAuth = async (user?: TestUser) => {
     .set('Accept', 'application/json')
     .set('Authorization', `Bearer ${auth_token}`);
 };
+
+export const deleteMailSlurperMails = async (user?: TestUser) => {
```

Comment:
Move the mailslurper calls in another file, e.g. mailslurper.rest.requests.ts.
The file name is misleading - you should have only the util to make rest requests there, not rest requests for a specific service / endpoint.